protoduneana/verticaldrift/singlehit/SingleHit_module.cc

```diff
@@ -594,7 +677,7 @@ void pdvdana::SingleHit::analyze(art::Event const& e)
   int PTSIsolated = vZIsolated.size();
   std::cout << " THERE IS " << PTSIsolated << " ISOLATED POINTS IN EVENT " << fEventID << std::endl;
```

Comment:
There are a lot of std::cout statements in here that look like they might be useful for debugging but will just clutter up logfiles. Is this module meant to be run in production? The message facility allows fcl steering of log levels and throttles many repeated messages, but sometimes I too prefer std::cout, if only to know that messages haven't been throttled. But ... Messages like these do not give the reader of a logfile any clue what piece of code wrote the message. If someone wants to understand what a particular message means, they have to look through all of DUNE code and LArSoft for the character string in a message. So please write the method name as part of the message, and make them optional.

Reply:
And there's a grammar error "IS ... POINTS"  (number mismatch).